script/tasks/diamondSyncWhitelistedAddresses.sh

```
@@ -96,73 +96,73 @@
         ATTEMPT=$((ATTEMPT + 1))
       done
 
-      echo "[error] Unable to fetch approved DEXs from $DIAMOND_ADDRESS on network $NETWORK after $MAX_ATTEMPTS_PER_SCRIPT_EXECUTION attempts."
+      echo "[error] Unable to fetch whitelisted addresses from $DIAMOND_ADDRESS on network $NETWORK after $MAX_ATTEMPTS_PER_SCRIPT_EXECUTION attempts."
       return 1 # Indicate failure
     }
 
-    # Fetch approved DEXs
-    DEXS=($(getApprovedDEXs))
+    # Fetch whitelisted addresses
+    WHITELISTED_ADDRESSES=($(getWhitelistedAddresses))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)